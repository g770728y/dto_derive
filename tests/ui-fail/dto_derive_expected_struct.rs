extern crate dto_derive;
use dto_derive::Dto;

#[derive(Debug, PartialEq, Dto)]
enum EnumDto { } //~ ERROR expected struct

fn main() { }
