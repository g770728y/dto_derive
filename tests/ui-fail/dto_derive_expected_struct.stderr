error: expected struct
 --> $DIR/dto_derive_expected_struct.rs:5:6
  |
5 | enum EnumDto { } //~ ERROR expected struct
  |      ^^^^^^^

error: aborting due to previous error

